flow-server/src/main/java/com/vaadin/flow/server/connect/VaadinConnectControllerConfiguration.java

```diff
@@ -137,4 +146,30 @@ public VaadinConnectAccessChecker accessChecker() {
     public ExplicitNullableTypeChecker typeChecker() {
         return new ExplicitNullableTypeChecker();
     }
+
+    /**
+     * Registers a {@link ObjectMapper} bean instance.
+     *
+     * @return the object mapper for endpoint.
+     */
+    @Bean
+    @Qualifier(VAADIN_ENDPOINT_MAPPER_BEAN_QUALIFIER)
+    ObjectMapper objectMapper(ApplicationContext context) {
```

Comment:
```suggestion
    public ObjectMapper vaadinEndpointMapper(ApplicationContext context) {
```
Change method name to reflect the `Bean` that's going to be created. 

Optional/Personal opinion: I would wrap the `ObjectMapper` in an object `VaadinConnectMapper#getObjectMapper` and return `VaadinConnectMapper` here as `Bean`, so it's easy to differentiate between the normal `ObjectMapper` and the Object Vaadin is using internally.